app/resources/forms.py

```diff
@@ -0,0 +1,32 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
+    StringField,
+    SubmitField
+)
+from wtforms.validators import InputRequired, Length
+
+
+class ResourceForm(Form):
+    name = StringField('Name', validators=[
```

Comment:
I don't think you actually need a description for the built in validators, I believe wtforms hooks it up.